### PR TITLE
🌐 Add Chinese translation for `docs/zh/docs/deployment/versions.md`

### PR DESCRIPTION
**Pull request by ponponon**
I added the Chinese translation about deployment, but only half of it is finished now. If the auditor thinks there is no problem, I will continue to complete the rest of the translation work

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+
+[dev-packages]
+
+[requires]
+python_version = "3.10"
```
Comment (tiangolo) [inline Pipfile:11]: Please remove this file.

File: docs/zh/docs/deployment/deta.md [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+* **Replication**: Handled by Deta, as part of their service.
+* **Memory**: Limit predefined by Deta, you could contact them to increase it.
+* **Previous steps before starting**: Not directly supported, you could make it work with their Cron system or additional scripts.
+
+!!! note
+    Deta is designed to make it easy (and free) to deploy simple applications quickly.
+
+    It can simplify several use cases, but at the same time, it doesn't support others, like using external databases (apart from Deta's own NoSQL database system), custom virtual machines, etc.
+
+    You can read more details in the <a href="https://docs.deta.sh/docs/micros/about/" class="external-link" target="_blank">Deta docs</a> to see if it's the right choice for you.
```
Comment (tiangolo) [inline docs/zh/docs/deployment/deta.md:258]: Please remove these extra files that are not part of the translation PR.

File: docs/zh/mkdocs.yml [hunk 3]
```diff
@@ -115,6 +115,15 @@ nav:
   - advanced/custom-response.md
   - advanced/response-cookies.md
   - advanced/wsgi.md
+- 部署:
+  - deployment/index.md
+  - deployment/versions.md
```
Comment (tiangolo) [inline docs/zh/mkdocs.yml:120]: Please include here only the translated files.

**Review by tiangolo — CHANGES_REQUESTED** [review]
Thanks for your effort!

I added a couple of comments to remove some files and lines that wouldn't be part of the PR. Thanks! :coffee:

**Comment by ponponon** [comment]
Great, I can't wait to see the new docs go live. I want to continue to finish the translation of the rest of the content.

**Comment by wdh99** [comment]
你要先把，除了文件`docs/zh/docs/deployment/versions.md`之外的所有修改复原。因为每个翻译的pull request只针对一个文件。

**Comment by tiangolo** [comment]
📝 Docs preview for commit 02f797239e9eaeef216af5599e7a0ce0a42f7c9b at: https://649a25cf5638651834126b7a--fastapi.netlify.app

**Comment by tiangolo** [comment]
As this PR had requested changes to be applied but has been inactive for a while, it's now going to be closed. But if there's anyone interested, feel free to create a new PR.

**Pull request closed without merging**